Image-only deploys (#3150)
astronomer/docs: astro/cli/astro-deploy.md

| `-n`, `--deployment-name` | The name of the Deployment to deploy to. Use as an alternative to `<deployment-id>`                                                                              | Any valid Deployment name                                        |
| `--description`           | A description for your code deploy. Descriptions appear in the Cloud UI in your Deployment's **Deploy History**                                                  | None                                                             |
| `-e`,`--env`              | Location of the file containing environment variables for pytests. By default, this is `.env`.                                                                   | Any valid filepath to an `.env` file                             |
| `-f`,`--force`            | Force deploy even if your project contains errors or uncommitted changes                                                                                         | None                                                             |

> | `--image`                 | If you have DAGs-only deploys enabled, use this flag to only deploy your Astro project image. When you use this option, it does not add update your DAGs folder. | None                                                             |

Reviewer: ```suggestion
| `--image`                 | If you have DAGs-only deploys enabled, use this flag to deploy only your Astro project image. When you use this option, your `dags` folder is not deployed to Astro. | None                                                             |
```

> | `--image`                 | If you have DAGs-only deploys enabled, use this flag to only deploy your Astro project image. When you use this option, it does not add update your DAGs folder. | None                                                             |

Reviewer 2: ```suggestion
| `--image`                 | If you have DAGs-only deploys enabled, use this flag to only deploy your Astro project image. When you use this option, it does not update your DAGs folder. | None                                                             |
```
